ow_plexil/src/plans/owlat_plans/OwlatPrepArm.plp

```diff
@@ -0,0 +1,25 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// This plan puts the arm into place above the ground plane to prepare for other
+// operations
```

Comment:
Curious how this is different from unstow, which is generally the "preparation" step before other arm operations.

Reply:
It is very similar to Unstow, except that it uses ARM_MOVE_CARTESIAN and brings it closer to the ground plane than Unstow probably should. I tried to keep them as faithful to the test procedures outlined in the user guide as possible, which calls for this extra step before a lot of the test sequences.

Maybe PrepArm is not a very accurate name, and something like OwlatPositionArm is better.

Reply:
That makes sense.  Yes, OwlatPositionArm is better.